mapping-engine-nodejs/README.md

```diff
@@ -48,34 +48,34 @@ Mapping adds device specific information to an Information Model. Since the repr
 
 To create a mapping go to your newly created model and press the **Create Mapping Spec** Button.
 
-![create mapping spec button](./images/create_mapping_key.png)
+![create mapping spec button](https://github.com/eclipse/vorto/raw/master/mapping-engine-nodejs/images/create_mapping_key.png)
```

Comment:
We should not put absolute paths here as it would not work on different branches. Please put relative paths. Take a look at other tutorial examples how to do that.

Reply:
The readme.md was already using relative paths, unfortunately the relative paths do not resolve for the package homepage in npm(https://www.npmjs.com/package/vorto-mapping-utility), hence the change.